### DISCUSSION TITLE
Why velero can not restore the resource if something updated

### DISCUSSION THREAD

**Discussion by storehubdeploy** [vmware-tanzu/velero#4747] [Community support Q&A]
I update the deployment and comfigmap, but velero can not restore to previous version
default:  could not restore, ConfigMap "nginx-config" already exists. Warning: the in-cluster version is different than the backed-up version.
              could not restore, Deployment "web" already exists. Warning: the in-cluster version is different than the backed-up version.

**Comment by sseago** [comment]
Currently Velero only restores resources that do not already exist in the cluster. If the resource exists and is different from what's backed up, the warning you saw is added to the restore. There is currently a feature enhancement under review which will make this configurable and allow an "update" policy which would make it possible for Velero to patch resources that already exist on restore. You can see that proposal and discussion here: https://github.com/vmware-tanzu/velero/pull/4613
